static/src/stylesheets/module/release-messages/_sign-in.scss

```diff
@@ -40,69 +44,57 @@ $sign-in-eb-logo: $gs-baseline * 3.5;
 }
 
 .site-message--sign-in-container {
+    padding: $sign-in-eb-padding-vertical 0;
+    font-size: 80%;
+    @media all and(min-height: $sign-in-eb-vertical-mq) {
+        font-size: 100%;
+    }
     .site-message--sign-in__header,
     .site-message--sign-in__body,
     .site-message--sign-in__buttons {
-        padding: $sign-in-eb-padding;
+        padding: 0 $sign-in-eb-padding;
     }
 }
 
 .site-message--sign-in__header {
-    @include fs-headline(3);
-    @media all and(min-height: 29em) {
-        @include fs-headline(4, true);
-    }
-    @media all and(min-height: 34em) {
-        @include fs-headline(5, true);
-    }
-    @media all and(min-height: 39em) {
-        @include fs-headline(6, true);
+    @include f-headline;
+    position: relative;
+    padding-right: $sign-in-eb-padding*3!important; /*close button*/
```

Comment:
Is there an alternative to using `!important` here?

Reply:
yes :(